new_platforms/vscode-extension/package.json

```diff
@@ -0,0 +1,83 @@
+{
+	"name": "msiot-vscode-openenclave",
+	"displayName": "Microsoft Open Enclave",
+	"description": "Develop, deploy, and debug your Microsoft Open Enclave",
+	"version": "1.0.2",
+	"publisher": "ms-iot",
+	"aiKey": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
```

Comment:
Should this be something more random?

Reply:
it is fake.  i'm not checking in the real one.

Reply:
i changed the checked in one to be: specifyt-here-alap-pins-ightskeyhere